ListApp/ListApp.Api/Utils/Constants.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using ListApp.Api.Models;
+
+namespace ListApp.Api.Utils
+{
+    public static class Constants
+    {
+        public static Guid Guid1 = Guid.Parse("00000000-0000-0000-0000-000000000000");
+        public static Guid Guid2 = Guid.Parse("00000000-0000-0000-0000-000000000001");
+        public static Guid Guid3 = Guid.Parse("00000000-0000-0000-0000-000000000002");
+        public static Guid NonExistingItemGuid = Guid.Parse("00000000-0000-0000-0000-000000000003");
+
+        public static IEnumerable<ListItem> MockListItems => new List<ListItem>
```

Comment:
Using an array would server better purpose here (since it is not permanent API and you want to access it by index (`MockListItems[0]` vs `MockListItems.ElementAt(0)`))